docs/designers-developers/developers/tutorials/devenv/readme.md

```diff
@@ -52,20 +75,32 @@ The WordPress [wp-env package](https://www.npmjs.com/package/@wordpress/env) let
 
 The `wp-env` package requires Docker to be installed. There are instructions available for installing Docker on [Windows 10 Pro](https://docs.docker.com/docker-for-windows/install/), [all other versions of Windows](https://docs.docker.com/toolbox/toolbox_install_windows/), [macOS](https://docs.docker.com/docker-for-mac/install/), and [Linux](https://docs.docker.com/v17.12/install/linux/docker-ce/ubuntu/#install-using-the-convenience-script).
 
-After confirming that the prerequisites are installed, install `wp-env` globally from the command-line using:
+After you have installed Docker, go ahead and install `wp-env` globally from the command-line using:
 
 ```sh
 npm -g install @wordpress/env
 ```
 
-This will be used in the next [Plugin step](wp-plugin.ms). To confirm it is installed and available, run:
+To confirm it is installed and available, run:
 
 ```sh
 wp-env --version
-> 1.4.0
+> 1.6.0
+```
+
+The `wp-env` script is used to create a Docker WordPress environment, to use start from your plugin directory—if you are following the create block tutorial, this would be in the generated directory.
+
```

Comment:
It is confusing going from working in terminal to suddenly mentioning the plugin directory.
Where is the plugin directory? We need a step that goes from checking the wp-env to locating the plugin directory and then running wp-env start. This needs clarification.

"...if you are following the create block tutorial, this would be in the generated directory."
This seems out of context. As the context is setting up a local dev environment.

We need a bridge going from checking the wp-env version to wp-env script.

(I added a similar comment in the below solved comment thread, but as it got hidden I am adding it again here.)

Reply:
I'm not sure setting up a dev environment with no context makes much sense. There is a reason you are setting up an environment, the most likely is for developing a plugin or theme. If you are following the create a block tutorial it provides the context [in that tutorial here](https://github.com/WordPress/gutenberg/blob/master/docs/designers-developers/developers/tutorials/create-block/wp-plugin.md).

Otherwise, you might of come to this page wanting to setup an environment to create a plugin, so you would have that in mind already.

Reply:
We has this introduction:
"The development environment setup guide is for setting up your local development environment for JavaScript development; creating plugins and tools for extending WordPress and the block editor."

This is how I understand it:
Setup a local WordPress development site for creating own plugin and/or contributing to WordPress/Gutenberg. (That is the context.) 

"If you are following the create a block tutorial it provides the context in that tutorial here."
As the doc we are working on right now is the very first tutorial for setting up a local dev site I find it confusing that someone would be following the next tutorial in the series so to speak. As the tutorial we have now would be a prerequisite for the block tutorial. Having a side path to tutorial 2 seems strange at this stage.